src/Runner.Worker/Handlers/NodeScriptActionHandler.cs

```diff
@@ -97,11 +97,11 @@ public async Task RunAsync(ActionRunStage stage)
                 workingDirectory = HostContext.GetDirectory(WellKnownDirectory.Work);
             }
 
-#if OS_OSX
+#if OS_OSX || OS_WINDOWS
             if (string.Equals(Data.NodeVersion, "node12", StringComparison.OrdinalIgnoreCase) &&
                 Constants.Runner.PlatformArchitecture.Equals(Constants.Architecture.Arm64))
             {
-                ExecutionContext.Output($"The node12 is not supported on macOS ARM64 platform. Use node16 instead.");
+                ExecutionContext.Output($"The node12 is not supported on the macOS or windows ARM64 platforms. Use node16 instead.");
```

Comment:
should we have 2 messages for each platform?
it might be weird to see a message about windows when I am on a macOS device.